src/Tribe/Linked_Posts.php

```diff
@@ -877,26 +877,32 @@ public function saved_linked_post_dropdown( $post_type, $current = null ) {
 			$data = array_values( (array) $options );
 		}
 
-		/*
+		$user_can_create = ( ! empty( $post_type_object->cap->create_posts ) && current_user_can( $post_type_object->cap->create_posts ) );
+		$allowed_creation = ( ! empty( $this->linked_post_types[ $post_type ]['allow_creation'] ) && $this->linked_post_types[ $post_type ]['allow_creation'] );
 
-		! empty( $post_type_object->cap->create_posts )
-		&& current_user_can( $post_type_object->cap->create_posts )
-		&& ! empty( $this->linked_post_types[ $post_type ]['allow_creation'] )
+		$placeholder = sprintf( esc_attr__( 'Select a %s', 'the-events-calendar' ), $singular_name );
+		if ( $user_can_create ) {
+			$placeholder = sprintf( esc_attr__( 'Create or Select a %s', 'the-events-calendar' ), $singular_name );
```

Comment:
I think we should swap Select and Create in this message so it matches the `$search_placeholder`. So:

```
__( 'Select or Create a %s', 'the-events-calendar' ),
```